packages/web3torrent/src/App.tsx

```diff
@@ -52,7 +52,9 @@ const App: React.FC = () => {
     return () => ({});
   }, []);
 
-  const ready = currentNetwork === requiredNetwork && initialized;
+  // TODO: re-enable this when we figure out the proper way to do this
+  //const ready = currentNetwork === requiredNetwork && initialized;
```

Comment:
It seems that we're not getting the correct chain id from metamask (at least in firefox). We seem to get back a `0` instead of `3141`. Mostly like due to changes to how `window.ethereum` works.

For now I just disabled the check.